### ISSUE TITLE
In search for ultimate bookmarks view

### ISSUE THREAD

**Issue by dzmitry-lahoda** [darekkay/static-marks#2]
I have tried `static-marks`. And it does not work for me.

I have 10K bookmarks. I use tags, I do not use directories to split them. So tagless view does not work for me.

Are you aware of any view with tag cloud and hierarchical tag faceting and local search build in? So I can export my knowledge graph for ease of search and use. 

I would like to share my knowledge with people by giving them links to tagged searches on my jamstack.

**Comment by darekkay** [comment]
You could probably import your tags as folders, but then you would get a lot of duplicate bookmarks. So yeah, Static Marks is probably not the right tool for you.

[Buku](https://github.com/jarun/Buku) is a similar project and it supports tags. [Pinboard](https://pinboard.in/) is another popular tag-based bookmark manager. Maybe one of those tools supports the workflow you need.

**Comment by dzmitry-lahoda** [comment]
Pinboard is closed source server. I would use only if community AGPL will be opensource. Once I have used Diigo, but when I did export - their export was totally broken. And they stopped to support Firefox(it was buggy and not priority).

Buku server is on python and there is no hosted solution, nor does it think via ipfs/data/syncthing/gun/orbit/git/crdt/p2p out of box.

Firefox bookmarks are open source (including server as I know). So I need export functional into static site.
